uint8 response_mode
uint32 response_ms
uint32 traj_id
bool run
uint32 vel
uint32 tol
uint8 frame_type
